backend/xivdpscalc/pps/sch_action.py

```diff
@@ -0,0 +1,39 @@
+""" Sch specific enums for simulation """
+
+from enum import Enum, auto
+from xivdpscalc.types import CastTime, ElapsedTime
+
+class SchAction(Enum):
+    """
+    Enum representing possible sch actions
+    """
+    BROIL3 = (0, 290, 2.5, True, False, 0)
+    RUIN2 = (1, 200, 0, True, False, 0)
+    AETHERFLOW = (2, 0, 0, False, False, 60)
+    ENERGYDRAIN = (3, 100, 0, False, False, 3)
+    BIOLYSIS = (4, 70, 0, True, True, 0)
```

Comment:
thinking about this for extensibility -- some dot actions deal damage on hit (example: dia, bard dots, chaos thrust, sonic break). biolysis does 0 on hit so this would technically be fine for now but if we can come up with a better representation to future-proof it that would be better